edenai_apis/utils/file_handling.py

```
@@ -27,59 +27,83 @@ def __init__(self):
+                ),
+            }
+
+            if file_size == 0:
+                raise Exception("File size is 0")
+
+            if force_file_create or file_size == -1 or file_size > self.SIZE_THRESHOLD:
+                # Download to file for large/unknown size files
+                try:
+                    response = await session.get(file_url, stream=True)
+                    response.raise_for_status()
+                except Exception as ex:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)